aiida/control/computer.py

```diff
@@ -0,0 +1,133 @@
+"""Manage computer objects with lazy loading of the db env"""
+from aiida.cmdline.utils.decorators import with_dbenv
+from aiida.utils.error_accumulator import ErrorAccumulator
+
+
+class ComputerBuilder(object):
+    """Build a computer with validation of attribute combinations"""
+
+    def __init__(self, **kwargs):
+        self._computer_spec = kwargs
```

Comment:
@ltalirz mentioned in the CodeBuilder that this would be safer instead:
```
for key, value in kwargs.items():
    self.__setattr__(key, value)
```
As it validates each keyword argument while setting it.

Reply:
ok